### ISSUE TITLE
Docker hub image for arm64 architecture fails when initializing it

### ISSUE THREAD

**Issue by danitico** [vespa-engine/vespa#26131]
**Describe the bug**
When running the docker image for arm64 version [8.128.22](https://hub.docker.com/layers/vespaengine/vespa/8.128.22/images/sha256-842f0741b33cb9998ed3353161cf4a597a5cb124f6ba75ab089e2d1a18fb53a8?context=explore), it constantly outputs the following [error](https://gist.github.com/danitico/be49902dc2b6d4dd5ef1094fa7867148). After that, it tries to restart but after all, it shows the same error.

**To Reproduce**
1. Download docker image for arm64 with version 8.128.22 and run it

**Expected behavior**
It should start correctly.

**Environment (please complete the following information):**
 - OS: MacOS
 - Infrastructure:self-hosted
 - Versions 8.128.22

**Vespa version**
8.128.22

**Comment by kkraune** [comment]
Yes, this is an error, I have reproduced it and we treat it as an incident - thanks for reporting! 

Please use vespaengine/vespa:8.128.22 until we have released a new image - sorry for the inconvenience

**Comment by aressem** [comment]
This is now fixed and `vespaengine/vespa:latest` resolves to a working `8.128.22`
